cruise.umple/src/Parser_Code.ump

```diff
@@ -1841,7 +1930,7 @@ class ErrorTypeSingleton
       int severityInt;
 
       if(code == null || severity == null || url == null || formatString == null)
-        throw new UmpleCompilerException("Error parsing error definitions (line " + p.getLineNumber()+")",null);
+        throw new RuntimeException("Error parsing error definitions (line " + p.getLineNumber()+")");
```

Comment:
I'd make this a better exception than RE, use `IllegalStateException` instead. Since that is what it is.